client-mr/src/main/java/org/apache/hadoop/mapreduce/MRRssUtils.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Tencent is pleased to support the open source community by making
+ * Firestorm-Spark remote shuffle server available.
+ *
+ * Copyright (C) 2021 THL A29 Limited, a Tencent company.  All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at
+ *
+ * https://opensource.org/licenses/Apache-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package org.apache.hadoop.mapreduce;
+
+import org.apache.hadoop.mapred.JobConf;
+
+import com.tencent.rss.client.api.ShuffleWriteClient;
+import com.tencent.rss.client.factory.ShuffleClientFactory;
+
+public class MRRssUtils {
+
+  // Class TaskAttemptId have two field id and mapId, mapId is 4 high byte,
+  // id is low 4 byte.
+  public static long convertTaskAttemptIdToLong(TaskAttemptID taskAttemptID) {
```

Comment:
taskId will be stored in bitmap at shuffle server, it should be designed to reduce memory cost. 
with mapId as high byte, I think it will waste more memory

Reply:
OK